blog/_posts/2017-11-01-zinc-blog-1.0.md

```diff
@@ -0,0 +1,280 @@
+algorithm front](https://github.com/sbt/zinc/issues/174). Finalizing these tasks
+was instrumental to measure how production-ready the algorithm was, and when a
+Zinc 1.0 release could happen.
+  
+As new issues started to be reported and more deficiencies to show up, I worked
+on other issues to stabilize the project. During this period I collaborated
+with the [Pants][] team at Twitter and other users building upon Zinc. This
+collaboration revolved around guaranteeing that Zinc satisfied other people's
+use cases aside from [sbt][]'s, and that end users would get the best
+incremental compilation experience possible.
+  
+All in all, this is a high-level summary of our contributions to Zinc 1.0:
+  
+* Improve correctness of the incremental algorithm in cases where it undercompiled.
+* Fixing bugs both in the Scala bridge and the Java incremental compilation.
+* Handle type-level programming more gracefully (for example, changes in type members).
+* Improve old Zinc APIs and create friendly APIs to complement missing public functionality.
+* Finish off the migration to Scala 2.12 and making Zinc ready for JDK8.
+* Add a protobuf-based analysis file with long term binary support.
+* Add features to Zinc that were missing and were considered important for Zinc's users (improvements to loggers and hooks for IDEs, for example).
+* Add JMH benchmarks to reason about the performance impact of incremental compiles.
+* Improve infrastructure to attract contributors:
+  * Speed up CI by 7x to have a fast turnaround time,
+  * Add more tests to the project,
+  * Add more documentation to the APIs, a README and a CONTRIBUTING guide; and,
+  * Add similar high-quality improvements to the build and scripted.
+  
+Find the [list of PRs here](https://github.com/sbt/zinc/pulls?utf8=%E2%9C%93&q=author%3Ajvican%20is%3Ap://github.com/sbt/zinc/pulls?utf8=%E2%9C%93&q=author%3Ajvican%20is%3Apr).
+  
+### External contributions
+
+Zinc 1.0.0 also has had significant contributions from the Scala open-source
+community, and I thank everyone involved to make it possible! The contributors
+since our involvement in the project have been [Stu Hood][], [Peiyu Wang][],
+[Krzysztof Romanowski][], [Wieslaw Popielarski][], [Peter Pan][], [Gregor
+Heine][], [Łukasz Indykiewicz][], [Allan Timothy Leong][], [Guillaume
+Martres][], [Wojciech Langiewicz][], [Krzysztof Borowski][], [Wojtek Pitula][],
+[knirski][], [Thierry Treyer][], [Grzegorz Kossakowski][], [Dale Wijnand][] and
+[Eugene Yokota][].
+
+[Martin Duhem] and I are happy that we met some of these contributors during our
+**[Scala Center sprees](https://github.com/scalacenter/sprees)** all over Europe
+and that they dared to submit pull requests to Zinc; while contributing to the
+project may at first sound daunting, you didn't give up and at the end your work
+had a clear impact on Scala developers. Congratulations!
+
+### A preview of Zinc 1.0: an example of a 7x speedup
```

Comment:
Maybe we can shuffle this section higher up in the article? I think people would be most excited by this info, so it might be good if it's one of the first things brought up in the article.